lib/chef/provisioning/aws_driver/driver.rb

```diff
@@ -360,6 +361,31 @@ def allocate_load_balancer(action_handler, lb_spec, lb_options, machine_specs)
       end
     end
 
+    # Compare two server certificates by casting them both to strings.
```

Comment:
Can you update this comment with `The parameters should either be a String containing the certificate ARN, or a IAM::ServerCertificate object.`